unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/Addressables/Tests/AddressablesEssentialsTests.cs

```diff
@@ -9,76 +9,73 @@
 using System.Threading.Tasks;
 using UnityEngine;
 
-namespace DCL.Tests
+[TestFixture][Category("ToFix")]
+public class AddressablesEssentialsTests
 {
-    [TestFixture]
-    public class AddressablesEssentialsTests
+    [Test]
+    public async Task EnsureEmbeddedEmotesEssentials()
     {
-        [Test]
-        public async Task EnsureEmbeddedEmotesEssentials()
-        {
-            // Arrange
-            var addressableProvider = new AddressableResourceProvider();
+        // Arrange
+        var addressableProvider = new AddressableResourceProvider();
 
-            //Act
-            EmbeddedEmotesSO asset = await addressableProvider.GetAddressable<EmbeddedEmotesSO>("EmbeddedEmotes.asset");
+        //Act
+        EmbeddedEmotesSO asset = await addressableProvider.GetAddressable<EmbeddedEmotesSO>("EmbeddedEmotes.asset");
 
-            //Assert
-            Assert.NotNull(asset);
+        //Assert
+        Assert.NotNull(asset);
 
-            //We check that the 31 embedded emotes are present
-            Assert.AreEqual(asset.emotes.Count(), 31);
+        //We check that the 31 embedded emotes are present
+        Assert.AreEqual(asset.emotes.Count(), 31);
 
-            //We validate just the first emote, wave
-            Assert.AreEqual(asset.emotes[0].id, "wave");
-            Assert.NotNull(asset.emotes[0].femaleAnimation);
-            Assert.NotNull(asset.emotes[0].maleAnimation);
-        }
+        //We validate just the first emote, wave
+        Assert.AreEqual(asset.emotes[0].id, "wave");
+        Assert.NotNull(asset.emotes[0].femaleAnimation);
+        Assert.NotNull(asset.emotes[0].maleAnimation);
+    }
 
-        [Test]
-        public async Task EnsureEssentials()
-        {
-            // Arrange
-            var addressableProvider = new AddressableResourceProvider();
+    [Test]
+    public async Task EnsureEssentials()
+    {
+        // Arrange
+        var addressableProvider = new AddressableResourceProvider();
 
-            //Act
-            var dabAnim = await addressableProvider.GetAddressable<AnimationClip>("dab.anim");
-            var tikAnim = await addressableProvider.GetAddressable<AnimationClip>("tik.anim");
-            var loadingScreen = await addressableProvider.GetAddressable<GameObject>("_LoadingScreen");
+        //Act
+        var dabAnim = await addressableProvider.GetAddressable<AnimationClip>("dab.anim");
+        var tikAnim = await addressableProvider.GetAddressable<AnimationClip>("tik.anim");
+        var loadingScreen = await addressableProvider.GetAddressable<GameObject>("_LoadingScreen");
```

Comment:
Loading screen left the addressables. I think we can remove this one, since its going to fail